### ISSUE TITLE
Stop relying on SSLv23_METHOD having SSLv3 in it

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7992]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7992|
|Type|enhancement|
|Created|2015-08-09 06:05:02Z|
|Branch|https://github.com/twisted/twisted/tree/sslv23method-7992|

```
[FAIL]
Traceback (most recent call last):
  File "/buildslave/fedora22-py2.7/Twisted/twisted/test/test_sslverify.py", line 1239, in test_SSLv23
    method=SSL.SSLv23_METHOD)))
  File "/buildslave/fedora22-py2.7/Twisted/twisted/trial/_synctest.py", line 437, in assertEqual
    super(_Assertions, self).assertEqual(first, second, msg)
  File "/usr/lib64/python2.7/unittest/case.py", line 553, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python2.7/unittest/case.py", line 836, in assertSetEqual
    self.fail(self._formatMessage(msg, standardMsg))
twisted.trial.unittest.FailTest: Items in the first set but not the second:
<ProtocolVersion=SSLv3>

twisted.test.test_sslverify.ProtocolVersionTests.test_SSLv23
```

<details><summary>Searchable metadata</summary>

```
trac-id__7992 7992
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_sslv23method_7992 branches/sslv23method-7992
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1439100302953929 1439100302953929
changetime__1445217443410268 1445217443410268
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45417]) Branching to sslv23method-7992.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Removed the test, since it's worthless.

https://buildbot.twistedmatrix.com/boxes-unsupported?branch=/branches/sslv23method-7992 shows green F22, the rest of the builders are green, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Looks good. I am ok with removing the tests as it looks like a test for the content of the SSL.SSLv23_METHOD constant.

Don't forget to add the news file... then merge :)

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45420]) Merge sslv23method-7992: Remove useless test that checks the contents of the SSLv23_METHOD constant

Author: hawkowl
Reviewer: adiroiban
Fixes: [#7992](https://github.com/twisted/twisted/issues/7992)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
